testing/cmake/test_export/CMakeLists.txt

```
@@ -16,7 +16,13 @@
 # THE SOFTWARE.
 
 cmake_minimum_required(VERSION 3.20.1)
-project(NVBenchTestExport CUDA CXX)
+project(NVBenchTestExport HIP CXX)
```

with this enable , that means, in a CUDA capable machine, we usually get errors since no cuda libraries found.

SInce HIP support CUDA and HIP, will it be better to have 

```suggestion
project(NVBenchTestExport CXX)
```

and set **COMPILER_HIP_COMPILER** or **COMPILER_CUDA_COMPILER** accordingly ?

---

We address this via RAPIDS-CMake https://github.com/ROCm/rapids-cmake/pull/5. The idea is to not break existing RAPIDS/CUDA build systems "too much".

This means you go this way:

1. (Only required until ROCm/rapids-cmake feature branch is merged): `export RAPIDS_CMAKE_BRANCH="feat/nvbench-delegation"`
2. You fetch the `ROCm/rapids-cmake` script
    * (REMARK: If you want to use CUDA anyway, you could also decide to fetch a `rapidsai/rapids-cmake` script here if you want to go with the latest and greatest. The `RAPIDS_CMAKE_BRANCH` environment variable won't be used in this case though.)

Then for the HIP backend (only with `ROCm/rapids-cmake` and the feature branch discussed in this comment):

3. You simply include `rapids-cmake/cpm/nvbench.cmake` and call `rapids_cpm_nvbench(...)`

While for the CUDA backend:

3. You set the `CUDA_BACKEND` CMake option to a positive CMake  boolean expression (`ON`/`on`/`1`/`TRUE`/`true`/`Yes`/`yes`/`y`)
4. You include `rapids-cmake/cpm/nvbench.cmake` and call `rapids_cpm_nvbench(...)`

---

@yiakwy-xpu-ml-framework-team  If you think there should be CUDA device support, please create an issue and we discuss this with management.

---

Suire I will do it. Currently I have some successful experiences in adding HIP support in existing CUDA cmake project . Of source this project is a fork from nvidia, it is reasonable to have HIP only support.